keyboards/sofle/keymaps/bastien-baloup_fr/config.h

```diff
@@ -0,0 +1,56 @@
+ /* Copyright 2022 Bastien Baloup
+  *
+  * This program is free software: you can redistribute it and/or modify
+  * it under the terms of the GNU General Public License as published by
+  * the Free Software Foundation, either version 2 of the License, or
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#pragma once
+
+// More lighting options in quantum/rgblight/rgblight.h
+#ifdef RGBLIGHT_ENABLE
+
+    // Disabling some of these is a good way to save flash space.
+//  #define RGBLIGHT_EFFECT_ALTERNATING     // 108
+//  #define RGBLIGHT_EFFECT_RGB_TEST        // 158
+//  #define RGBLIGHT_EFFECT_RAINBOW_MOOD    // 160
+//  #define RGBLIGHT_EFFECT_STATIC_GRADIENT // 168
+    #define RGBLIGHT_EFFECT_RAINBOW_SWIRL   // 192
+//  #define RGBLIGHT_EFFECT_BREATHING       // 348
+//  #define RGBLIGHT_EFFECT_KNIGHT          // 336
+//  #define RGBLIGHT_EFFECT_SNAKE           // 406
+//  #define RGBLIGHT_EFFECT_CHRISTMAS       // 508
+//  #define RGBLIGHT_EFFECT_TWINKLE         // 1156
```

Comment:
Could the byte counts be removed?  These are not very accurate, and differ greatly depending on the compiler version, some other features being enabled, etc.